src/main/java/io/codelex/arithmetic/exercises/Exercise5.java

```diff
@@ -7,15 +7,15 @@
     public static void main(String[] args) {
         Scanner input = new Scanner(System.in);
 
-        System.out.println("I'm thinking of a number between 1-100.  Try to guess it.");
+        System.out.println("I'm thinking of a number between 1-100. Try to guess it.");
         int userInput = input.nextInt();
 
         int comparableNumber = calculateRandomNumber();
 
         if (userInput < comparableNumber) {
-            System.out.println("Sorry, you are too low.  I was thinking of " + comparableNumber + ".");
+            System.out.println("Sorry, you are too low. I was thinking of " + comparableNumber + ".");
         } else if (userInput > comparableNumber) {
-            System.out.println("Sorry, you are too high.  I was thinking of " + comparableNumber + ".");
+            System.out.println("Sorry, you are too high. I was thinking of " + comparableNumber + ".");
         } else  {
             System.out.println("You guessed it!  What are the odds?!?");
```

Comment:
pretty low :laughing: bet šis uzdevusm laikam kaut kā noklīdis no arithmetic :smile:

Reply:
Nu, iedūrās man acī... 😆 Bet, jā, šim te nevajadzēja būt.